### DISCUSSION TITLE
Pointing a subdomain to an external server (not working)

### DISCUSSION THREAD

**Discussion by lipceanu** [vercel/community#4918] [Help]
### Summary

My domain was initially purchased via Google domains, so Google domains is currently pointing towards Vercel's ns servers. Now as we need to create an external application on our subdomain: "office.alfa-point.com", I now need the subdomain to point onto an external VPS that we are using for that application. 

I can access the server via the IP Address, however the subdomain doesn't work. 

In the Vercel's documentation they mention adding A and CNAME record, however that is not possible because for the main domain, being "alfa-point.com" the CNAME record already exists and it doesn't allow creation of that.

Any suggestions? 

### Steps to Reproduce

n/a

**Comment by Georgian** [comment]
If anyone stumbles upon this, my problem was a firewal (`ufw`) misconfiguration.

Pinging the subdomain (configured on Vercel with a simple `A` record) worked but browser wasn't serving the files.

Change UFW rules from `Nginx HTTP` to `Nginx Full`
